common/alignment/alignment-en.html

```diff
@@ -0,0 +1,98 @@
+---
+{
+	"title": "GCWeb Alignment example",
+	"language": "en",
+	"altLangPage": "alignment-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-06-19"
+}
+---
+<div class="wb-prettify all-pre"></div>
+
+<h2>Horizontal Alignment</h2>
+<h3> Left aligned <code>text-left</code></h3>
+<p class="text-left">Use to align text to the left.</p>
+<h4>Code</h4>
+<pre><code>&lt;p class="<strong>text-left</strong>"&gt;Use to align text to the left.&lt;/p&gt;</code></pre>
+
+<h3> Right aligned <code>text-right</code></h3>
+<p class="text-right">Use to align text to the right.</p>
+<h4>Code</h4>
+<pre><code>&lt;p class="<strong>text-right</strong>"&gt;Use to align text to the right.&lt;/p&gt;</code></pre>
+
+<h3> Center <code>text-center</code></h3>
+<p class="text-center">Use to align text to the center.</p>
+<h4>Code</h4>
+<pre><code>&lt;p class="<strong>text-center</strong>"&gt;Use to align text to the center.&lt;/p&gt;</code></pre>
+
+<h3> Center content blocks <code>center-block</code></h3>
+<div class="panel panel-default">
+	<div class="panel-body">
+	<img src="https://placehold.it/140x140" class="img-rounded center-block" alt="A generic square placeholder image with rounded corners">
+	</div>
+</div>
+<h4>Code</h4>
+<pre><code>&lt;img <strong>class=&quot;center-block&quot;</strong> alt=&quot;...&quot; src=&quot;x.jpg&quot; /&gt;</code></pre>
+
+<h3> Left aligned from small view <code>text-sm-left</code></h3>
+<p class="text-right text-sm-left">Left-aligned in small view and over</p>
+<h4>Code</h4>
+<pre><code>&lt;p class="text-right <strong>text-sm-left</strong>"&gt;Left-aligned in small view and over&lt;/p&gt;</code></pre>
+
+<h3> Right aligned from small view <code>text-sm-right</code></h3>
+<p class="text-sm-right">Right-aligned in small view and over</p>
+<h4>Code</h4>
+<pre><code>&lt;p class="<strong>text-sm-right</strong>"&gt;Right-aligned in small view and over&lt;/p&gt;</code></pre>
+
+<h2> Vertical alignment </h2>
+<table>
+	<thead>
+		<tr>
+			<th>Class</th>
+			<th>Example</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<th><code>align-top</code>
```

Comment:
Add the descriptive text like we did in French. And do the same for the other rows.

```suggestion
			<th>Top (<code>align-top</code>)
```